packages/privacy-toolset/README.md

```diff
@@ -0,0 +1,37 @@
+# Components
+
+A library of tools React components designed to unify privacy mechanisms in Automattic products.
+
+## Installation
+
+Install the toolset.
+
+```bash
+yarn add @automattic/privacy-toolset
+```
+
+## Usage
+
+```jsx
+// import the component you wish to use
+import { CookieBanner } from '@automattic/privacy-toolset';
+
+const Component = () => (
+	<>
+		<CookieBanner content={ ...args } onAccept={ fn } />
+	</>
+);
+```
+
+## Development Workflow
+
+This package is developed as part of the Calypso monorepo. Run `yarn`
+in the root of the repository to get the required `devDependencies`.
+
+### Tests
+
+N/A
```

Comment:
I've left this section when we'll start adding more logic to here (this lib is created to have both react component and helper functions). I'm not sure if we should test such presentational react components? Or do you see a good testing case here?

Reply:
On components such as these, I've seen test cases written to make sure the React state is handled correctly when certain actions are taken. Could be useful here! Not going to fight to say it's pertinent though

Reply:
I would echo that some tests would be useful since this is a new package that will have some wider application. I think it could be useful to test:
- Output of simpleConsent shows as expected on initial render
- customConsent shows when clicking the "Customize" control
- The onAccept handler prop receives expected/  properly typed input from the component for both the "Accept All" action and a customized selection of preferences.

Reply:
Thanks! I've implemented tests based on snapshots and listening to onAccept calls.